### ISSUE TITLE
Feature request: create new empty session and enable drag/drop open tabs

### ISSUE THREAD

**Issue by davidjade** [ReDEnergy/SessionSync#56]
The end goal for me is, I want to save a few tabs into a new session - not all open tabs. So I would like to create a new saved session and put just the open tabs I select into that new session.

These two suggestions to enable that go hand-in-hand.

First, make the creation of a new empty session more discoverable. Right now you can do that by right-clicking on an existing session but I only found that by accident. I would suggest making it a higher-level toolbar button, etc... btw, I can only get this by right-clicking on an existing session - clicking on surrounding white space does not give me that option.

Then make these new empty session more useful by allowing drag/drop of tabs in the current active list into saved sessions.

With these two feature I could create new sessions with just the open tabs I want.

**Comment by ReDEnergy** [comment]
Thanks for the feedback.
Regarding the requests...
1. make the creation of a new empty session more discoverable
I was idiot and somehow I removed that option in the last release... Not sure how but somehow I did that because it was possible to right click on an empty space and create a new session. I will put that back of course.
2. Not sure why I didn't thought about this, but it's a wonderful idea and I will definitely implement it also.
